vkcs/compute/resource_vkcs_compute_instance.go

```diff
@@ -252,9 +255,9 @@ func ResourceComputeInstance() *schema.Resource {
 						"boot_index": {
 							Type:        schema.TypeInt,
 							Optional:    true,
-							Default:     -1,
+							Computed:    true,
 							ForceNew:    true,
-							Description: "The boot index of the volume. It defaults to -1. Changing this creates a new server. _note_ You must set the boot index to 0 for one of the block devices if more than one is defined.",
+							Description: "The boot index of the volume. Changing this creates a new server. _note_ You must set the boot index to 0 for one of the block devices if more than one is defined.",
```

Comment:
Do not we need to mention that 1) it is default to 0 for the only device 2) it is default to -1 for multiple devices?

Reply:
Added.